src/Storage/src/Models/AuthorizationCode.cs

```diff
@@ -108,5 +108,13 @@ public class AuthorizationCode
         /// The code challenge method
         /// </value>
         public string CodeChallengeMethod { get; set; }
+
+        /// <summary>
+        /// Gets or sets properties
+        /// </summary>
+        /// <value>
+        /// The properties
+        /// </value>
+        public IDictionary<string, object> Properties { get; set; } = new Dictionary<string, object>();
     }
```

Comment:
Why `object` in the `Dictionary<string, object>`? I worry about the values not round tripping.

Reply:
done